packages/bsky/src/index.ts

```diff
@@ -161,10 +168,12 @@ export class BskyAppView {
     await events.once(server, 'listening')
     const { port } = server.address() as AddressInfo
     this.ctx.cfg.assignPort(port)
+    await this.ctx.featureGates.start()
```

Comment:
I think we'll want to move this up to before the server is listening / fielding requests.